packages/datadog_flutter_plugin/android/datadog_version.toml

```diff
@@ -0,0 +1,6 @@
+[versions]
+datadog-android = "1+"
+
+[libraries]
+datadog-sdk-android = { module = "com.datadoghq:dd-sdk-android", version.ref = "datadog-android" }
+datadog-sdk-android-ndk = { module = "com.datadoghq:dd-sdk-android-ndk", version.ref = "datadog.android" }
```

Comment:
```suggestion
datadog-sdk-android-ndk = { module = "com.datadoghq:dd-sdk-android-ndk", version.ref = "datadog-android" }
```

Reply:
because of the implicit conversion to `.`:

> When declaring aliases, it’s worth noting that any of the -, _ and . characters can be used as separators, but the generated catalog will have all normalized to .

But it is still worth to have the consistency.